pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -62,6 +62,26 @@ def test_agg_ser_multi_key(df):
     tm.assert_series_equal(results, expected)
 
 
+def test_agg_different_partials():
+    # for issue 28570
```

Comment:
nitpick: can you write this `GH#28570`?  makes it easier to grep for issue numbers